src/modules/ordersTable/containers/OrdersTableWidget/hooks/useOrdersTableList.ts

```diff
@@ -32,7 +32,7 @@ export function useOrdersTableList(allOrders: Order[]): OrdersTableList {
       (acc, order) => {
         if (PENDING_STATES.includes(order.status)) {
           acc.pending.push(order)
-        } else {
+        } else if (!order.composableCowInfo?.isVirtualPart) {
```

Comment:
Can you use the utils to identify the order?

Reply:
Fixed